packages/flame/lib/src/components/router_component.dart

```diff
@@ -255,9 +239,9 @@ class RouterComponent extends Component {
   }
 
   /// Local method to bypass [pop]'s assert
-  void _popWithoutMinimumAssert() {
+  void _replaceRoute(Route nextRoute) {
```

Comment:
`_removeTopRoute`? This method doesn't actually replace anything.

Reply:
Done